.github/workflows/report-robot-results.yml

```diff
@@ -0,0 +1,49 @@
+name: "Collect Robot Results"
+
+# we have multiple workflows - this helps to distinguish for them
+run-name: "${{ github.event.workflow_run.head_branch }} - Report Robot Results"
+
+on:
+  workflow_run:
+    workflows: ["Build & Test"] # runs after build and test workflow
+    types:
+      - completed
+
+permissions:
+  contents: read
+  actions: read
+  checks: write
+
+#
+# see https://github.com/dorny/test-reporter?tab=readme-ov-file#recommended-setup-for-public-repositories
+#
+jobs:
+  #
+  # Collect and upload sonar report with coverage generated by the Build & Test workflow
+  #
+  robot-report:
+    name: #robot-Report
+    runs-on: ubuntu-latest
+    if: github.event.workflow_run.conclusion == 'success'
+    steps:
+
+      # Download jacoco overall coverage from build & test output
+      - name: Download - Robot Results
+        uses: actions/download-artifact@v4
+        with:
+          github-token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
Should it maybe be the `BOT_ACCESS_TOKEN`?

Reply:
oops ... fixed thx